README.md

```diff
@@ -1,2 +1,16 @@
-# mlkube
+# MLKube
 Machine Learning Infrastructure built on Kubernetes
+
+Kubernetes makes it possible for anyone to build container-based infrastructure.
+MLKube, an open source project, leverages the extensibility of Kubernetes to enable running machine learning infrastructure on top of Kubernetes.
```

Comment:
A syntax error: s/enable running/ enable the running of Or enable us to run ?

Reply:
done.